osl/interface_linux.go

```diff
@@ -17,15 +17,16 @@ import (
 type IfaceOption func(i *nwIface)
 
 type nwIface struct {
-	srcName     string
-	dstName     string
-	master      string
-	dstMaster   string
-	address     *net.IPNet
-	addressIPv6 *net.IPNet
-	routes      []*net.IPNet
-	bridge      bool
-	ns          *networkNamespace
+	srcName        string
+	dstName        string
+	master         string
+	dstMaster      string
+	address        *net.IPNet
+	addressIPv6    *net.IPNet
+	routes         []*net.IPNet
+	bridge         bool
+	ns             *networkNamespace
+	addressAliases []*net.IPNet
```

Comment:
True, never realized it was a Cisco terminology. Agree, let's stick to the Linux terminology. Thanks.